google/cloud/storage/internal/hash_validator_test.cc

```diff
@@ -224,7 +228,7 @@ TEST(CreateHashValidator, Read_Null) {
       CreateHashValidator(ReadObjectRangeRequest("test-bucket", "test-object")
                               .set_multiple_options(DisableCrc32cChecksum(true),
                                                     DisableMD5Hash(true)));
-  validator->Update("The quick brown fox jumps over the lazy dog");
+  validator->Update("The quick brown fox jumps over the lazy dog", 43);
```

Comment:
It looks like there are still 8 instances of the explicit-count variety.

Reply:
Thanks! Fixed.